bot/constants.py

```diff
@@ -24,6 +24,10 @@
 LOG_FILE = pathlib.Path("log/gurkbot.log")
 
 
+if TEST_GUILDS := os.getenv("TEST_GUILDS"):
+    TEST_GUILDS = [int(x.strip()) for x in TEST_GUILDS.split(",")]
```

Comment:
```suggestion
    TEST_GUILDS = [int(x) for x in TEST_GUILDS.split(",")]
```
This will suffice since `int()` strips whitespace.

Reply:
https://github.com/gurkult/gurkbot/pull/167/commits/d381cbdf863fa9b0ce718450d33554f664d98ba1